### DISCUSSION TITLE
How to put focus on the preview panel without the mouse?

### DISCUSSION THREAD

**Discussion by andrewakl** [hluk/CopyQ#1819] [General]
When i press the F7 key, the Preview panel shows up.  I tried pressing the Tab key to put the focus on the Preview panel so that i can scroll down with the cursor keys to view the entire contents.  However, the Tab key didn't behave as i expected.

I couldn't find any option to assign a keyboard shortcut to put focus on the Preview panel.  I prefer to not have to move the mouse into the Preview panel and clicking the mouse button to put the focus into the Preview panel.

**Comment by hluk** [comment]
>  I tried pressing the Tab key to put the focus on the Preview panel so that i can scroll down with the cursor keys to view the entire contents. However, the Tab key didn't behave as i expected.

How does it behave? For me, the preview panel is focused after pressing Tab (from item list).

Are you using CopyQ 5.0.0 (the latest version)?

**Reply by andrewakl** [reply]
Hi.
Yes... i'm using CopyQ v5.0.0 running on Windows 10 Enterprise (version: 20H2) as well as Windows 10 Home (version 21H1).

This is the behavior i see:
![copyq-preview-panel](https://user-images.githubusercontent.com/59714327/141827893-695ccf76-03d8-4c57-8e82-a5573f9e8cd8.gif)

The preview panel appears to be detached from the main window (item list)

**Reply by hluk** [reply]
The the preview dock needs to be attached to the main window. Tab key does not work in multiple windows/docks.

**Reply by hluk** [reply]
I will fix it in #1826.
